src/chocolatey/infrastructure.app/nuget/NuGetEndpointResources.cs

```diff
@@ -105,21 +107,21 @@ public PackageSearchResource SearchResource
 
         public SourceRepository Source { get; private set; }
 
-        public static NuGetEndpointResources GetResourcesBySource(SourceRepository source)
+        public static NuGetEndpointResources GetResourcesBySource(SourceRepository source, ChocolateySourceCacheContext cacheContext)

```

Comment:
This was missed initially, we need to add this as an overload and make the old method obsolete.